backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -237,4 +239,10 @@ export class ListingDto extends OmitType(Listing, [
   @Type(() => ListingUtilitiesDto)
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   utilities?: ListingUtilitiesDto
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
```

Comment:
What does the each:true do here?

Reply:
Oop that's for fields that are an array definitely just a copy 🍝  ty!